lib/versioning/npm/range.ts

```diff
@@ -113,6 +113,39 @@ export function getNewValue({
           ? `>= ${toVersion}`
           : `>=${toVersion}`;
       }
+    } else {
+      const newRange = parseRange(currentValue);
+      const versions = newRange.map((x) => {
+        const subRange = x.semver;
+        const bumpedSubRange = getNewValue({
+          currentValue: subRange,
+          rangeStrategy: 'bump',
+          fromVersion,
+          toVersion,
+        });
+        if (satisfies(toVersion, bumpedSubRange)) {
+          return bumpedSubRange;
+        }
+        if (satisfies(toVersion, subRange)) {
+          return subRange;
+        }
+        let incremented;
+        if (x.patch !== '0') {
+          incremented = increment(toVersion, 'patch');
+        } else if (x.minor !== '0') {
+          incremented = increment(toVersion, 'minor');
+        } else {
+          incremented = increment(toVersion, 'major');
+        }
+        const replaced = getNewValue({
+          currentValue: subRange,
+          rangeStrategy: 'replace',
+          fromVersion,
+          toVersion: incremented,
+        });
+        return satisfies(toVersion, replaced) ? replaced : null;
+      });
+      return versions.filter((x) => x !== null && x !== '').join(', ');
```

Comment:
I think commas aren't valid in semver ranges, so this is incorrect to have in npm versioning